site/content/en/docs/getting_started/overview.md

```diff
@@ -0,0 +1,202 @@
+---
+title: 'CVAT Overview'
+linkTitle: 'Introduction to CVAT'
+weight: 1
+description: 'CVAT: The open-source tool for image and video annotation and data labeling'
+---
+
+Machine learning systems often struggle due to
+poor-quality data. Without effective tools,
+improving a model can be tough and inefficient.
+
+[**CVAT.ai**](https://www.cvat.ai/) **is a** versatile **tool** for
+**annotating images and videos**, serving the computer
+vision community worldwide.
+
+Our goal is to help developers, businesses,
+and organizations globally by using a Data-centric AI approach.
+
+CVAT offers two versions:
+
+- [**CVAT Cloud**](https://app.cvat.ai/): Start online with CVAT,
+  **available for free**. You can also choose a [**subscription**](https://www.cvat.ai/pricing/cloud)
+  for **unlimited data**, **organizational tools**, **auto-annotations**, and [**more**](https://www.cvat.ai/post/cvat-ai-pricing-plans-choosing-the-right-plan-for-your-needs).
+
+- **Self-hosted CVAT** (forever Free of charge): Follow the [**Self-hosted Installation Guide**](/docs/administration/basics/installation/)
+  for setup. We provide [**Enterprise-level support**](https://www.cvat.ai/pricing/on-prem)
+  for this version, **including premium features** like **SSO**, **LDAP**, advanced integrations with
+  [**Roboflow and HuggingFace**](https://www.cvat.ai/post/integrating-hugging-face-and-roboflow-models),
+  and **advanced analytics**. We also offer **professional training** and **24-hour SLA support**.
+
+
+See:
+
+- [Tools and formats](#tools-and-formats)
+  - [Supported formats](#supported-formats)
+  - [Annotation tools](#annotation-tools)
+- [Automated labeling](#automated-labeling)
+- [Useful links](#useful-links)
+- [Integrations](#integrations)
+- [License Information](#license-information)
+- [Get in touch](#get-in-touch)
+
+## Tools and formats
+
+CVAT stands as a comprehensive
+tool for image and video annotation,
+essential for various computer vision tasks.
+It emphasizes user-friendliness,
+adaptability, and compatibility with a range of formats and tools.
+
+### Supported formats
+
+CVAT's supports the following formats:
+
+- **For 3D**: `.pcd`, `.bin`
+- **For image**: everything supported by the Python Pillow library,
+  including formats like `JPEG`, `PNG`, `BMP`, `GIG`, `PPM`and `TIFF`.
+- **For video**: all formats, supported by ffmpeg, including `MP4`, `AVI`, and `MOV`.
+
+For annotation export and import formats, see
+[**Export annotations and data from CVAT**](/docs/manual/advanced/formats/)
+
+### Annotation tools
+
+CVAT offers a wide array of annotation tools,
+each catering to different aspects of image and video labeling:
+
+- **3D Object Annotation**: For annotating objects in three dimensions,
```

Comment:
I would recommend to insert here just links
`[3D Object Annotation](relative path to documentation)`

I don't think that such "obvious" description makes sense.

Reply:
@nmanovic I've added links but would like to save the descriptions. Descriptions give small advice that users sometimes ask in Discord for example.  I will elaborate on them a bit more in some kind of next iteration, this page will need a couple of updates shortly (for example when I will finish updating all the tools docs).